sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -918,10 +919,16 @@ public static void endSession() {
   /**
    * Gets the current active transaction or span.
    *
-   * @return the active span or null when no active transaction is running
+   * @return the active span or null when no active transaction is running. In case of
+   *     globalHubMode=true, always the active transaction is returned, rather than the last active
+   *     span.
    */
   public static @Nullable ISpan getSpan() {
-    return getCurrentHub().getSpan();
+    if (globalHubMode && Platform.isAndroid()) {
```

Comment:
but this is still not the only thing we want, right? In integrations like okhttp, where we use `hub.getTransaction` we should use either `getTransaction` or `getSpan` depending on whether it's running on Android or not